lua/options.lua

```diff
@@ -26,6 +26,7 @@ vim.opt.termguicolors = true
 -- how many spaces should a tab be
 vim.opt.tabstop = 2
 
--- TODO: Describe leader key
+-- The "Leader key" is a way of extending the power of VIM's shortcuts by using sequences of keys to perform a command. 
+-- The default leader key is backslash. Therefore, if you have a map of <Leader>Q, you can perform that action by typing \Q.
```

Comment:
**Suggestion -** Add help docs reference

I am trying to put as many help doc references as I can in this project to get users comfortable with reading them. Can you add a `:help <leader>` comment here?
``